services/wallet/collectibles/service.go

```diff
@@ -491,24 +499,26 @@ func (s *Service) notifyCommunityCollectiblesReceived(ownedCollectibles OwnedCol
 	}
 
 	groups := make(map[CollectibleGroup]Collectible)
-	for i, collectible := range communityCollectibles {
+	for _, collectible := range communityCollectibles {
+		txHash := ""
 		for key, value := range hashMap {
 			if key.Same(&collectible.ID) {
-				communityCollectibles[i].LatestTxHash = value.Hex()
+				collectible.LatestTxHash = value.TxID.Hex()
```

Comment:
do you need to edit the element in the array, or a copy of the one in the current iteration of the loop?

Reply:
ahh seems like you're adding the elements to another map and not reusing the array, so it looks like the change is good!

Reply:
lol another workaround required because of using ERC721 instead of ERC1155...
@alaibe